portfolio/src/main/webapp/script.js

```diff
@@ -15,14 +15,38 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
+function addFunction() {
   const greetings =
       ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
 
   // Pick a random greeting.
   const greeting = greetings[Math.floor(Math.random() * greetings.length)];
 
   // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
+  const greetingContainer = document.getElementById('function-container');
```

Comment:
Nit: consider changing the name back to greeting container to be more descriptive of what's being returned.